.github/workflows/publish.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         printf -- "---\n:rubygems_api_key: ${RUBYGEMS_API_KEY}\n" > $HOME/.gem/credentials
         ./gradlew --stacktrace publishMavenPublicationToMavenCentralRepository gemPush
       env:
-        ORG_GRADLE_PROJECT_ossrhUsername: ${{ secrets.OSSRH_USERNAME }}
+        ORG_GRADLE_PROJECT_ossrhUsername: ${{ vars.OSSRH_USERNAME }}
```

Comment:
This change means that use `OSSRH_USERNAME` Variables instead of Secret. correct?
<img width="754" alt="スクリーンショット 2023-10-20 17 46 31" src="https://github.com/embulk/embulk-parser-msgpack/assets/767650/f929072e-f0de-423b-958d-5bc61b942ba8">

Reply:
Yes. GitHub had only Secrets, no Variables in the past.